packages/cli/src/commands/hydrogen/deploy.ts

```diff
@@ -77,13 +78,14 @@ export default class Deploy extends Command {
     }),
     'no-json-output': Flags.boolean({
       description:
-        'Prevents the command from creating a JSON file containing the deployment URL (in CI environments).',
+        'Prevents the command from creating a JSON file containing the deployment URL in CI environments.',
       required: false,
       default: false,
     }),
     token: Flags.string({
       char: 't',
-      description: 'Oxygen deployment token',
+      description:
+        "Oxygen deployment token. Defaults to the linked storefront's token if available.",
```

Comment:
Not sure if here is the place to mention it, but this only defaults to the linked storefront outside CI environments, and is a required flag within one.